code/game/objects/items/weaponry.dm

```diff
@@ -925,7 +925,7 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	item_state = "classic_baton"
 	lefthand_file = 'icons/mob/inhands/equipment/security_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/security_righthand.dmi'
-	force = 10
+	force = 14
```

Comment:
Billy club shouldn't be buffed. It fills a niche a a practical tool, not a weapon.

Reply:
This buff was intended after a long talk with "Nerf Lover" on discord. The majority of people that have joined this conversation assumed it had a force of 15 and were genuinely confused when it was mentioned it only had a force of 10 at the time.

So, after some talking, we decided it would be a good idea to increase it to 14. To at least side grade the knife as a blunt choice and even after me increasing it to 14, there was a minor backlash of it being too low compared to the knife.

Reply:
I said that because as it stands this will make the billy club essentially dead content if it doesn't get upped in some form. I did say that with the assumption the bonus damage to structures would be reduced in turn but that doesn't seem to have been the case. The ideal for me would be the billy club on green, knife on blue and above. But the whole point of the PR is to add a lethal option to blue so ¯\\\_(ツ)_/¯ and before its said I know the billy club is lethal, but a blunt weapon _seems_ a whole lot less than a knife does

Reply:
Yes, better to separate out these kinds of changes.